setup.cfg

```
@@ -32,7 +32,8 @@ install_requires =
 tests_require=
     pytest
     coverage
-    h-matchers>=1.0, <1.1
+    h-matchers==1.2.5
```

```suggestion
    h-matchers>=1.2.5
```

I wonder what this was all about.